packages/authgear-core/src/container.ts

```diff
@@ -156,7 +156,7 @@ export abstract class BaseContainer<T extends BaseAPIClient> {
   }
 
   /**
-   * @public
+   * @internal
    */
   getAccessToken(): string | undefined {
     return this.accessToken;
```

Comment:
Why don't we remove this function?

Reply:
API client is using this function to access the access token during fetch